HousingRepairsSchedulingApi/Gateways/DrsAppointmentGateway.cs

```diff
@@ -67,14 +67,15 @@ public class DrsAppointmentGateway : IAppointmentsGateway
         }
 
         public async Task<string> BookAppointment(string bookingReference, string sorCode, string locationId,
-            DateTime startDateTime, DateTime endDateTime)
+            DateTime startDateTime, DateTime endDateTime, string orderComments)
         {
             Guard.Against.NullOrWhiteSpace(bookingReference, nameof(bookingReference));
             Guard.Against.NullOrWhiteSpace(sorCode, nameof(sorCode));
             Guard.Against.NullOrWhiteSpace(locationId, nameof(locationId));
+            Guard.Against.NullOrWhiteSpace(orderComments, nameof(orderComments));
             Guard.Against.OutOfRange(endDateTime, nameof(endDateTime), startDateTime, DateTime.MaxValue);
 
```

Comment:
This should validate the `orderComments` parameter, i.e. not null and length is less than 255, but (I think) whitespace could be ok.

Check if an empty description is valid in the DRS request as the [previous code](https://github.com/Newarkandsherwood/HousingRepairsSchedulingApi/pull/17/files#diff-79bd046138cf9b29869c6c1a4e1be8b99aab7095df5aea775cbc67d59b754230L91) had a single space (` `) character and I suspect DRS may reject the request with an empty description - please check and a validation accordingly.